Gemfile

```diff
@@ -97,6 +97,12 @@ gem 'sass'
 gem 'sass-rails'
 gem 'coffee-rails'
 gem 'uglifier', '>=1.0.3'
+# Provide a JS runtime to execjs without needing to
+# have node, bin, or similar installed on the relevant
```

Comment:
Should "bin" be "bun"?

Reply:
hilariously, no: https://bun.sh/

Reply:
So what's "bin"?

Reply:
oh, shit, i misread your comment, and my comment. fml.